source/Government.cpp

```diff
@@ -407,6 +446,29 @@ string Government::Fine(PlayerInfo &player, int scan, const Ship *target, double
 
 
 
+bool Government::Condemns(const Outfit *outfit) const
+{
+	const auto isAtrocity = atrocities.find(outfit);
+	bool found = isAtrocity != atrocities.cend();
+	return (found && isAtrocity->second) || (!found && outfit->Get("atrocity") > 0.);
+}
+
+
+
+int Government::Fines(const Outfit *outfit) const
+{
+	// If this government doesn't fine anything it won't fine this outfit.
+	if(!fine)
+		return 0;
+
+	for(const auto &it : illegals)
+		if(it.first == outfit)
+			return it.second;
+	return outfit->Get("illegal");
+}
```

Comment:
Is there any point where you only use one of these? Why not just have the Fines method return negative if(Condemns)?
Does a government that doesn't fine still have the possibility of condemning for an outfit?

Reply:
Condemning is for atrocities, so yes you can not fine but condemn, condemns = they shoot you on sight, often, meaning that a fine doesnt matter in that case.
If you're on a planet you actually just die because of it its quite brutal.